src/main/java/org/quiltmc/config/impl/util/ConfigsImpl.java

```diff
@@ -40,6 +39,13 @@ public static void put(String familyId, Config config) {
 		CONFIGS.computeIfAbsent(familyId, id -> new TreeMap<>()).put(config.id(), config);
 	}
 
+	/**
+	 * Not intended to be used for anything other than testing.
```

Comment:
`@org.jetbrains.annotations.TestOnly` is exactly for this

Reply:
oh neat!